tools/log_example_mri.py

```diff
@@ -50,34 +67,74 @@ def main():
     log = sunlog.log_file_to_list(args.logfile)
 
     # plot log data
-    steps, times, vals = sunlog.get_history(log, 'h',
-                                            step_range=args.step_range,
-                                            time_range=args.time_range)
+    steps_s, times_s, vals_s = sunlog.get_history(
+        log,
+        "h",
+        step_range=args.step_range,
+        time_range=args.time_range,
+        group_by_level=True,
+    )
+
+    # get data for error test failures
+    steps_etf, times_etf, vals_etf = sunlog.get_history(
```

Comment:
Do you want to split up the failures by level as well?

Reply:
I tried that to start, but I though it created too many legend entries. I found it generally easy to distinguish failures of the different levels.